RingPublishingTracking/src/main/java/com/ringpublishing/tracking/internal/audio/AudioEventParam.kt

```diff
@@ -0,0 +1,21 @@
+package com.ringpublishing.tracking.internal.audio
+
+internal enum class AudioEventParam(val text: String) {
+    SELECTED_ELEMENT_NAME("VE"),
+    EVENT_TYPE("RT"),
+    CONTENT_ID("PMU"),
+    DURATION("VT"),
+    CURRENT_TIME("VP"),
+    DATA("VC"),
```

Comment:
audioParameters? zamiast data
ja to w ogole wole jak to jest od razu tam w faktorce, bez tego enuma ale ok